client/src/components/button/index.js

```diff
@@ -0,0 +1,14 @@
+import React, { Component } from 'react';
+import '../../App.css';
```

Comment:
I think this won't need to import App.css, as the App component will always be present, and it imports that stylesheet.

Reply:
Done :+1: